### ISSUE TITLE
[Question] No Matching Distribution Found for libtorrent

### ISSUE THREAD

**Issue by fazaghifari** [DLR-RM/stable-baselines3#1363]
### ❓ Question

I'm having trouble installing stable-baselines3[extra] on my Mac. I'm not sure if I missed installing any dependencies, I just followed the instruction `!pip install 'stable-baselines3[extra]'`

My machine configuration:
* Macbook M1 Pro
* Monterey 12.6.3
* Miniconda environment with Python 3.9.6 and Pip 22.3.1

```
Collecting stable-baselines3[extra]
  Downloading stable_baselines3-1.7.0-py3-none-any.whl (171 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 171.8/171.8 kB 4.6 MB/s eta 0:00:00
Requirement already satisfied: cloudpickle in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (2.2.1)
Requirement already satisfied: numpy in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (1.24.2)
Collecting gym==0.21
  Using cached gym-0.21.0-py3-none-any.whl
Collecting pandas
  Downloading pandas-1.5.3-cp39-cp39-macosx_11_0_arm64.whl (11.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.0/11.0 MB 10.2 MB/s eta 0:00:00
Requirement already satisfied: torch>=1.11 in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (1.13.1)
Requirement already satisfied: matplotlib in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (3.7.1)
Collecting importlib-metadata~=4.13
  Downloading importlib_metadata-4.13.0-py3-none-any.whl (23 kB)
Collecting tensorboard>=2.9.1
  Downloading tensorboard-2.12.0-py3-none-any.whl (5.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.6/5.6 MB 11.1 MB/s eta 0:00:00
Collecting psutil
  Using cached psutil-5.9.4-cp38-abi3-macosx_11_0_arm64.whl (244 kB)
Collecting rich
  Downloading rich-13.3.2-py3-none-any.whl (238 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 238.7/238.7 kB 9.6 MB/s eta 0:00:00
Requirement already satisfied: pillow in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (9.4.0)
Requirement already satisfied: tqdm in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (4.65.0)
Requirement already satisfied: opencv-python in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from stable-baselines3[extra]) (4.7.0.72)
Collecting autorom[accept-rom-license]~=0.4.2
  Downloading AutoROM-0.4.2-py3-none-any.whl (16 kB)
Collecting ale-py==0.7.4
  Downloading ale_py-0.7.4-cp39-cp39-macosx_11_0_arm64.whl (1.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 11.7 MB/s eta 0:00:00
Requirement already satisfied: importlib-resources in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from ale-py==0.7.4->stable-baselines3[extra]) (5.12.0)
Requirement already satisfied: requests in /Users/fazaghifari/miniconda3/envs/rlplaygroundenv/lib/python3.9/site-packages (from autorom[accept-rom-license]~=0.4.2->stable-baselines3[extra]) (2.28.2)
Collecting click
  Using cached click-8.1.3-py3-none-any.whl (96 kB)
Collecting AutoROM.accept-rom-license
  Downloading AutoROM.accept-rom-license-0.5.5.tar.gz (22 kB)
  Installing build dependencies ... error
  error: subprocess-exited-with-error
  
  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [12 lines of output]
      Ignoring importlib-resources: markers 'python_version < "3.9"' don't match your environment
      Collecting setuptools
        Downloading setuptools-67.5.1-py3-none-any.whl (1.1 MB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 10.4 MB/s eta 0:00:00
      Collecting wheel
        Using cached wheel-0.38.4-py3-none-any.whl (36 kB)
      Collecting requests
        Using cached requests-2.28.2-py3-none-any.whl (62 kB)
      Collecting tqdm
        Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
      ERROR: Could not find a version that satisfies the requirement libtorrent (from versions: none)
      ERROR: No matching distribution found for libtorrent
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip
```

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is minimal and working
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
Duplicate of https://github.com/Farama-Foundation/AutoROM/issues/20

See https://github.com/Farama-Foundation/AutoROM/issues/39 for alternative.

**Comment by araffin** [comment]
If you don't use Atari games, you can also omit the "extra".

**Comment by fazaghifari** [comment]
My bad, didn't realize that was the AutoROM. Thanks!

**Comment by fazaghifari** [comment]
For those who encounter same problem on Mac M1:
* Install AutoROM 0.4.2 explicitly `pip install AutoROM.accept-rom-license==0.4.2` as suggested in https://github.com/Farama-Foundation/AutoROM/issues/20
* Install stable-baselines3[extra] `pip install 'stable-baselines3[extra]'`

**Comment by jjshoots** [comment]
Hi everyone, all the problems should be fixed with AutoROM v0.6.0:
```
pip3 install AutoROM>=0.6.0
```
We don't recommend anyone use any version of AutoROM prior to 0.6.0 anymore.

Cheers!
